Gemfile

```diff
@@ -33,6 +33,7 @@ gem 'rails-i18n', '~> 0.7.3'
 gem 'recaptcha', '~> 0.4.0', :require => 'recaptcha/rails'
 gem 'rmagick', '~> 2.14.0'
 gem 'ruby-msg', '~> 1.5.0',  :git => 'https://github.com/mysociety/ruby-msg.git', :ref => 'f9f928ed76c024b4bc3a08bc1a59beb62df36663'
+gem 'sass', '3.4.21'
```

Comment:
Isn't this a dependency of `compass-rails`? Shouldn't we let that handle the version of the `sass` gem?